### ISSUE TITLE
Cloudworkshop storage account container name change

### ISSUE THREAD

**Issue by jtuliani** [microsoft/MCW-Line-of-business-application-migration#15]
The name of the storage container in the cloudworkshop storage account, which contains files used by this HOL, needs to be changed from azure-migration to line-of-business-application-migration

**Comment by DawnmarieDesJardins** [comment]
@paulopsgility 
Adding for visibility

**Comment by jtuliani** [comment]
Fixed in 'may-2019-fixes' branch
Once this branch is merged into Master we can delete the existing 'azure-migration' container. Let's keep this issue open until that is done.

**Comment by DawnmarieDesJardins** [comment]
Resolved with may 2019 fixes PR.
